CMake/kwiver-setup-python.cmake

```diff
@@ -0,0 +1,118 @@
+###
+# Finds the python binaries, libraries, inclue, and site-packages paths
```

Comment:
Please add some documentation about the variables that are exported or intended for use outside of this script, here. E.g. kwiver_python_subdir

Reply:
Nothing is imported, but I'll add a message about the exports.